db/migrate/20231120094146_add_users_franceconnect_openid_sub_index.rb

```diff
@@ -0,0 +1,5 @@
+class AddUsersFranceconnectOpenidSubIndex < ActiveRecord::Migration[7.0]
+  def change
+    add_index :users, :franceconnect_openid_sub
```

Comment:
je pense qu'il est préférable d'utiliser un index partiel pour éviter d'indexer les NULL (ultra majoritaires) et ainsi garder des index petits.

```suggestion
    add_index :users, :franceconnect_openid_sub, where: "franceconnect_openid_sub IS NOT NULL"
```

J'avais gardé cet article sous la main pour en parler à l'équipe (il parle des index partiels mais aussi d'autres techniques qui pourraient nous être utiles) : 
https://hakibenita.com/postgresql-unused-index-size